nutils/evaluable.py

```diff
@@ -1094,28 +1094,29 @@ def evalf(evalargs):
         return weights
 
 
-class Normal(Array):
+class Orthonormal(Array):
     'normal'
```

Comment:
I think we can improve a bit on the documentation. Something like: Orthonormalize the vector to/w.r.t the basis.

Reply:
changed it to "make a vector orthonormal to a subspace"